crates/rust-analyzer/src/cli/load_cargo.rs

```diff
@@ -82,6 +82,8 @@ pub fn load_workspace(
     log::debug!("crate graph: {:?}", crate_graph);
     let host =
         load_crate_graph(crate_graph, project_folders.source_root_config, &mut vfs, &receiver);
+
+    host.analysis().prime_caches(|_| {}).expect("Failed to prefill caches");
```

Comment:
The method is used in the `load_workspace_at` only and that one is used in benchmarks, analysis stats, cli diagnostics and ssr.

To me it looks fine to enable priming unconditionally, but maybe it's better to add a boolean parameter instead?